Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.3.0
+
+WORKDIR /usr/local/src/
+
+ADD . /usr/local/src/
+RUN cd /usr/local/src/
```

Comment:
Not necessary, because of the `WORKDIR`.

I recommend this approach, which allows you to skip bundler when the bundle hasn't changed:

```
WORKDIR /usr/local/src
ADD Gemfile Gemfile.lock /usr/local/src/
RUN bundle install --without development
ADD . /usr/local/src/
CMD bundle exec ...
```